app/controllers/pages_controller.rb

```diff
@@ -39,6 +39,9 @@ def show
   end
 
   def follow_up
+    if !params[:member_id].present? && recognized_member.try(:id).present?
+      return redirect_to follow_up_member_facing_page_path(@page.id, member_id: recognized_member.id)
+    end
```

Comment:
@NealJMD Please correct me if Im wrong here: if I understand this code correctly, you are doing a redirect to the same page the user is already in order to add the `member_id` when we only know the `akid`. 

Is there any reason we cannot do this using javascript from the follow-up page view? I'm thinking we could add an endpoint that retrieves the member_id for the current `recognized_member` (via akid or cookies). Doing this would help us avoid having this redirect.